source/missions/missions/pilot/mission.sqf

```diff
@@ -99,4 +116,4 @@ if (vehicle _pilot != _pilot) then {
 deleteVehicle _pilot;
 
 // Give cookies  (bonus & notifications)
-[20, _mission_name ] execVM "missions\mission_score.sqf";
\ No newline at end of file
+[20, _mission_name ] execVM "missions\mission_score.sqf";
```

Comment:
just a note, no need to act on it. I have noticed that you are removing the empty line at the end of the files, that is a habit I have from C code, in my previous job there was a hard rule of ending files with a new line (still don't know why)